docs/administration/data/data-migration.md

```diff
@@ -29,6 +29,14 @@ The data migration tools are not suitable for every imaginable scenario. In thes
 1. To move your entire Octopus Server from a self-hosted installation to Octopus Cloud, see [migrating from self-hosted to Octopus Cloud](/docs/octopus-cloud/migrations.md).
 1. To move your entire Octopus Cloud instance to a self-hosted Octopus Server please [contact our support team](mailto:support@octopus.com) for advice and assistance.
 
+## Unsupported scenarios
```

Comment:
Converting the section to a callout (problem not warning - problem is red, warning yellow) and removing the headings as they won't validate inside a callout:
```suggestion
:::problem
**Unsupported scenarios**
```